### PR TITLE
Escape missing fields

### PR DESCRIPTION
**Pull request by afbora**

### CODE REVIEW

File: kirby/config/blocks/markdown/markdown.php [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <?php /** @var \Kirby\Cms\Block $block */ ?>
-<?= $block->text()->kt();
+<?= $block->text()->html()->kt();
```
Comment (afbora) [inline kirby/config/blocks/markdown/markdown.php:2]: I'm not sure. But this does not damage markdown tags but escape HTML tags successfully.

Comment (bastianallgeier) [reply kirby/config/blocks/markdown/markdown.php:2]: There's an issue with `<http://link.com>` and mixed HTML in markdown.

Comment (afbora) [reply kirby/config/blocks/markdown/markdown.php:2]: So let's revert

Comment (afbora) [reply kirby/config/blocks/markdown/markdown.php]: Reverted

**Comment by afbora** [comment]
I tried all block fields. I only saw issue in image and markdown. I reverted it as it can use HTML tag inside the markdown.

**Comment by lukasbestle** [comment]
To be honest I wonder why the `alt` field of the image block in particular is unescaped while all other fields of all other blocks are. Where does this escaping on the content level happen and why doesn't it apply to `alt`?

**Comment by bastianallgeier** [comment]
The alt attribute is managed by a regular text input. The other fields are Writers and they handle HTML sanitization already via ProseMirror.

**Comment by lukasbestle** [comment]
Ah, that makes sense! 👍

I'm currently checking each snippet again.

**Review by lukasbestle — COMMENTED** [review]
I found two more possible issues in the image block: External image URLs and the link field can contain a valid URL that ends with `"><script>...`. So I added escaping to these as well.

Looks good now apart from what we are discussing in Discord.

**Review by lukasbestle — APPROVED** [review]

**Pull request merged**